module/VuFind/src/VuFind/ILS/Driver/SierraRest.php

```diff
@@ -2568,44 +2616,35 @@ protected function getItemStatus($item)
      * Determine whether an item is holdable
      *
      * @param array $item Item from Sierra
+     * @param array $bib  Bib record from Sierra
      *
      * @return bool
+     *
+     * @SuppressWarnings(PHPMD.UnusedFormalParameter)
```

Comment:
It's for any local needs because the now-extinct `itemHoldAllowed` method also had it. There was a bad title hold specific check for bib level in `itemHoldAllowed`. But now I realized that removing it made it impossible to control item hold in this respect, so I reintroduced the check but added a possibility to configure it separately. This makes the new functionality back-compatible configuration-wise.

Reply:
Thanks! But now I think you need to remove this SuppressWarnings annotation (looks like it's still there as of this writing).